python/tests/integration/core/test_core_stream.py

```diff
@@ -26,6 +26,8 @@
 SCHEMA = TransactionType.SCHEMA
 
 
+# TODO: Understand if we need this test. If yes, rewrite it in BDDs for all the drivers.
```

Comment:
I'd delete it for now, since this test isn't actually doing anything??
Although, I think we should have a BDD to test the behaviour of queries:

1. read queries can be opened and iterated concurrently & interleaved freely
2. write and schema queries cancel previous read and write queries from responding their answers

So as soon as you run a write of schema query, previous iterators will eventually be interrupted (didn't manage to make it "eager", youll either be able to finish the answers in the buffer and it's OK, or you eventually get an error in those streams.

Commit, rollback, and close actually do the same things!

Reply:
Added todos, removed the test